<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-13T13:26:43 IST" name="Default suite" finished-at="2024-10-13T13:27:09 IST" duration-ms="26152">
    <groups>
    </groups>
    <test started-at="2024-10-13T13:26:43 IST" name="Default test" finished-at="2024-10-13T13:27:09 IST" duration-ms="26152">
      <class name="test_cases.verify_add_department">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:test_cases.verify_add_department@7c37508a]" started-at="2024-10-13T13:26:44 IST" name="launchApplication" finished-at="2024-10-13T13:26:59 IST" duration-ms="15229" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="addDepartment(java.util.HashMap)[pri:0, instance:test_cases.verify_add_department@7c37508a]" started-at="2024-10-13T13:26:59 IST" name="addDepartment" data-provider="getTestData" finished-at="2024-10-13T13:27:09 IST" duration-ms="9651" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{newDepartment=Top Executives, existingDepartment=Users Without Department, user=Cena, John}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [(0)] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [(0)] but found []
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at base_pages.Departments_Page.verifyDepartmentUserCount(Departments_Page.java:70)
at test_cases.verify_add_department.addDepartment(verify_add_department.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addDepartment -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:test_cases.verify_add_department@7c37508a]" started-at="2024-10-13T13:27:09 IST" name="tearDown" finished-at="2024-10-13T13:27:09 IST" duration-ms="557" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test_cases.verify_add_department -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
